src/sinks/util/http.rs

```diff
@@ -22,6 +22,7 @@ use tracing_tower::{InstrumentableService, InstrumentedService};
 
 pub type Response = hyper::Response<Bytes>;
 pub type Error = hyper::Error;
+pub type HttpsClient = hyper::Client<HttpsConnector<HttpConnector<Resolver>>>;
```

Comment:
ah perfect, I thought I added this a while back and was just looking for it in the kube watchclient PR! So happy you added this :)